article/sections/_2_graph_importance.tex

```diff
@@ -101,7 +88,8 @@ \section{Perils of disregard}
 
 Figure \ref{fig:IND_GRAPH} illustrates the DAG where all explanatory variables are independent of each other for each utility equation.
 This DAG represent a scenario where an analyst only modifies variables in their dataset that are directly impacted by the proposed intervention, without considering the different relationships between these variables and other explanatory variables in the dataset.
-Similarly, Figure \ref{fig:DA_causal_2} through \ref{fig:BIKE_causal_2} illustrate the causal graphs with ``dependent'' explanatory variables.
+Similarly, Figure \ref{fig:DA_causal_2} through \ref{fig:SR3_causal_2} illustrate the causal graphs with ``dependent'' explanatory variables.
+This DAG represents a scenario where an analyst considers the relationships between all variables and modifies all children nodes of the variable directly impacted by the proposed intervention.
 For our purposes, we assume that these DAGs represent the ``true'' data generating process, where several variables affect each other.
 Each of these graphs is based on the utility function of each mode in the multinomial logit model specified in \citet{brathwaite_asymmetric}.
```

Comment:
```suggestion
To illustrate the problem with this approach, consider the case where the "true" data generating process that reflects the dependencies between the covariates is as shown in Figures \ref{fig:DA_causal_2} through \ref{fig:SR3_causal_2}. 
Under this generative model, intervening on one variable would also result in changes to other variables that are dependent on it.
The goal of our simulation exercise is to show that ignoring the true generative model of the data would result in arbitrarily biased treatment effects, even if the analyst knows the true outcome model.
```

Reply:
lol @hassan-obeid, please use `` '' instead of " " when writing latex.

Reply:
```suggestion
To illustrate the problem with this approach, consider the case where the ``true'' data generating process that reflects the dependencies between the covariates is as shown in Figures \ref{fig:DA_causal_2} through \ref{fig:SR3_causal_2}. 
Under this generative model, intervening on one variable would also result in changes to other variables that are dependent on it.
The goal of our simulation exercise is to show that ignoring the true generative model of the data would result in arbitrarily biased treatment effects, even if the analyst knows the true outcome model.
```

Reply:
I fixed it here @timothyb0912